scipy/stats/_qmc.py

```diff
@@ -940,56 +940,61 @@ def reset(self):
-
-    Centered discrepancy-based design shows better space filling robustness
-    toward 2D and 3D subprojections. Distance-based design shows better space
-    filling but less robustness to subprojections [2]_.
+    Optimize the sample by doing random permutations of coordinates to lower
+    the centered discrepancy [1]_. The resulting sample is still a LHS.
 
     Parameters
     ----------
     d : int
         Dimension of the parameter space.
-    start_design : array_like (n, d), optional
-        Initial design of experiment to optimize. `OrthogonalLatinHypercube`
+    start_sample : array_like (n, d), optional
+        Initial sample to optimize. `LatinHypercube`
         is used to generate a first design otherwise.
     niter : int, optional
-        Number of iterations to perform. Default is 1.
-    method : callable ``f(func, x0, bounds)``, optional
-        Optimization function used to search new samples. Default to
-        *basinhopping* optimization.
+        Number of iterations of optimization to perform. Default is 1.
     seed : {None, int, `numpy.random.Generator`}, optional
         If `seed` is None the `numpy.random.Generator` singleton is used.
         If `seed` is an int, a new ``Generator`` instance is used,
         seeded with `seed`.
         If `seed` is already a ``Generator`` instance then that instance is
         used.
 
+    Notes
+    -----
+    The specified optimization `method` is used to select a new set of
+    permutations to perform. If `method` is None,
+    `~scipy.optimize.dual_annealing` optimization is used. `niter` set of
+    optimizations are performed.
+
+    During the optimization, the best design is constantly update. This is
+    known as
+    Enhanced Stochastic Evolutionary algorithm (ESE) in the literature.
+
+    Centered discrepancy-based design shows better space filling robustness
+    toward 2D and 3D subprojections. Distance-based design shows better space
+    filling but less robustness to subprojections [2]_.
+
     References
     ----------
-    .. [1] Fang et al. Design and modeling for computer experiments,
-       Computer Science and Data Analysis Series, 2006.
+    .. [1] Jin et al., "An efficient algorithm for constructing optimal design
+       of computer experiments."
+       Journal of Statistical Planning and Inference, 2005.
     .. [2] Damblin et al., "Numerical studies of space filling designs:
-       optimization of Latin Hypercube Samples and subprojection properties",
+       optimization of Latin Hypercube Samples and subprojection properties."
        Journal of Simulation, 2013.
 
     Examples
     --------
     Generate samples from an optimal design.
 
     >>> from scipy.stats import qmc
-    >>> sampler = qmc.OptimalDesign(d=2, seed=12345)
+    >>> sampler = qmc.OptimalLatinHypercube(d=2)
     >>> sample = sampler.random(n=5)
     >>> sample
-    array([[0.84052691, 0.53664833],
+    array([[0.84052691, 0.53664833],  # random
```

Comment:
Aren't we using `default_rng()` which, under the hood, gets an explicit seed so that it's not random?

Reply:
Same